src/pages/layer-2.tsx

```diff
@@ -0,0 +1,702 @@
+    },
+    {
+      emoji: ":hammer_and_wrench:",
+      title: t("layer-2-expand-use-cases-title"),
+      description: t("layer-2-expand-use-cases-description"),
+    },
+  ]
+
+  const rollupCards = [
+    {
+      image: OptimisticRollupImage,
+      title: t("layer-2-optimistic-rollups-title"),
+      description: t("layer-2-optimistic-rollups-description"),
+      childSentence: t("layer-2-optimistic-rollups-childSentance"),
+      childLink: "/developers/docs/scaling/optimistic-rollups/",
+    },
+    {
+      image: ZKRollupImage,
+      title: t("layer-2-zk-rollups-title"),
+      description: t("layer-2-zk-rollups-description"),
+      childSentence: t("layer-2-zk-rollups-childSentance"),
+      childLink: "/developers/docs/scaling/zk-rollups/",
+    },
+  ]
+
+  const toolsData = {
+    information: [
+      {
+        title: "L2BEAT",
+        description: t("layer-2-tools-l2beat-description"),
+        link: "https://l2beat.com",
+        image: L2BEATImage,
+        alt: "L2BEAT",
+      },
+      {
+        title: "L2 Fees",
+        description: t("layer-2-tools-l2fees-description"),
+        link: "https://l2fees.info",
+        image: DogeImage,
+        alt: "L2 Fees",
+      },
+      {
+        title: "Chainlist",
+        description: t("layer-2-tools-chainlist-description"),
+        link: "https://chainlist.org",
+        image: DogeImage,
+        alt: "Chainlist",
+      },
+    ],
+    walletManagers: [
+      {
+        title: "Zapper",
+        description: t("layer-2-tools-zapper-description"),
+        link: "https://zapper.fi/",
+        image: ZapperImage,
+        alt: "Zapper",
+      },
+      {
+        title: "Zerion",
+        description: t("layer-2-tools-zerion-description"),
+        link: "https://zerion.io",
+        image: ZerionImage,
+        alt: "Zerion",
+      },
+      {
+        title: "DeBank",
+        description: t("layer-2-tools-debank-description"),
+        link: "https://debank.com",
+        image: DebankImage,
+        alt: "DeBank",
+      },
+    ],
+  }
+  
+  return (
+    <Flex flexDirection="column" alignItems="center">
+      <PageMetadata
+        title={t("layer-2-hero-title")}
+        description={t("layer-2-metadata-description")}
+      />
+      {/* Hero Section */}
+      <HubHero {...heroContent} />
+      {/* What is Layer 2 Section */}
+      <ContentBox id="what-is-layer-2">
+        <TwoColumnContent>
+          <Box flex="50%">
+            <SectionHeading>
+              {t("layer-2-what-is-layer-2-title")}
+            </SectionHeading>
+            <Text>
+              <Translation id="page-layer-2:layer-2-what-is-layer-2-1" />
+            </Text>
+            <Text>
+              {t("layer-2-what-is-layer-2-2")}
+            </Text>
+          </Box>
+          <Box flex="50%">
+            <Image
+              src={WhatIsEthereumImage}
+              alt=""
+              maxH="400px"
+              objectFit="contain"
```

Comment:
We need to constrain this width image as well.
```suggestion
              width={500}
              style={{ objectFit: "contain" }}
```